connectors/uplynk/src/main/java/com/theoplayer/android/connector/uplynk/internal/network/PreplayInternalResponse.kt

```diff
@@ -33,5 +34,10 @@ internal data class MinimalPreplayResponse(
 
 internal class PreplayInternalResponse(val body: String, private val json: Json) {
```

Comment:
Nitpick: let's rename this to PreplayInternal**Vod**Response to match the public classes.

Reply:
Thanks, renamed that